### PR TITLE
 spawn to increase stacksize for test

### PR DESCRIPTION
**Pull request by rchaser53**
related: #3473

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -165,6 +165,8 @@ pub struct FormatReport {
     non_formatted_ranges: Vec<(usize, usize)>,
 }
 
+unsafe impl Send for FormatReport {}
```
Comment (scampi) [inline src/lib.rs]: out of curiosity, why is the unsafe needed here ?

Comment (rchaser53) [reply src/lib.rs]: Send cannot be implemented without unsafe

Comment (topecongiro) [reply src/lib.rs]: `FormatReport` is part of public API, so I prefer not to change it. Also, `FormatReport` should not be marked as `Send` as it uses `Rc`. You need to replace it with `Arc`.

**Comment by rchaser53** [comment]
Ok. I also think your source is better than mine.

**Pull request closed without merging**